app/frontend/wallet/shelley/shelley-js-crypto-provider.ts

```diff
@@ -42,7 +42,7 @@ CryptoProviderParams): Promise<CryptoProvider> => {
 
   const isHwWallet = () => false
 
-  const getWalletName = () => 'Mnemonic'
+  const getWalletName = (): WalletName => WalletName.MNEMONIC
```

Comment:
Btw is not the return type `WalletName.MNEMONIC`?
The same question for the usage below.
I am just curious if it is not good to be explicit it the return type itself is explicit.
What do you think?

Reply:
Fixed. Forced it into a commit where it belonged.